### DISCUSSION TITLE
ERROR: initializing node: starting node: open /home/miner1/lotus/api: is a directory

### DISCUSSION THREAD

**Discussion by cheeseburgercod** [filecoin-project/lotus#7768] [General Q&A]
The machine was working fine, installed lotus, downloaded parameters, synced the chain. Rebooted machine and now lotus daemon will not run. Displays this error.

ERROR: initializing node: starting node: open /home/miner1/lotus/api: is a directory

**Comment by cheeseburgercod** [comment]
I deleted the lotus directory & .lotus directory, cloned the github repo again, downloaded parameters again, and still the same error message

ERROR: initializing node: starting node: open /home/miner1/lotus/api: is a directory

**Comment by LexLuthr** [comment]
@cheeseburgercod Do you still need help with this error or were you able to resolve it?

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new update for more than 7 days from the author.
